index.html

```diff
@@ -31,6 +31,100 @@
+                    <li><a href=""><button id="skill">javaScript</button></a></li>
+                </ul>
+                <button class="normal-button">See Project</button>
+            </div>
+
+
+        </div>
+        <div class="card-works">
+            <img class="porfolio-card1" src="./images/Snapshoot Portfolio 2.png" alt="">
+        <div class="project-desc">
+            <p class="project-title">Multi-Post Stories</p>
+            <div class="frame2">
+                <h3>CANOPY</h3>
+                <img src="./images/Counter.png" alt="">
+                <h4>Back End Dev</h4>
+                <img src="./images/Counter.png" alt="">
+                <h4>2015</h4> 
+            </div>
+             
+            <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tag">
+                <li><a href=""><button id="skill">html</button></a></li>
+                <li><a href=""><button id="skill">css</button></a></li>
+                <li><a href=""><button id="skill">javaScript</button></a></li>
+            </ul>
+            <button class="normal-button">See Project</button>
+        </div>
+
+
+    </div>
+    <div class="card-works">
+        <img class="porfolio-card1" src="./images/Snapshoot Portfolio 3.png" alt="">
+    <div class="project-desc">
+        <p class="project-title">Tonic</p>
+        <div class="frame2">
+            <h3>CANOPY</h3>
+            <img src="./images/Counter.png" alt="">
+            <h4>Back End Dev</h4>
+            <img src="./images/Counter.png" alt="">
+            <h4>2015</h4> 
+        </div>
+         
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag">
+            <li><a href=""><button id="skill">html</button></a></li>
+            <li><a href=""><button id="skill">css</button></a></li>
+            <li><a href=""><button id="skill">javaScript</button></a></li>
+        </ul>
+        <button class="normal-button">See Project</button>
+    </div>
+
+
+</div>
+<div class="card-works">
+    <img class="porfolio-card1" src="./images/Snapshoot Portfolio 4.png" alt="">
+<div class="project-desc">
+    <p class="project-title">Multi-Post Stories</p>
+    <div class="frame2">
+        <h3>CANOPY</h3>
+        <img src="./images/Counter.png" alt="">
+        <h4>Back End Dev</h4>
+        <img src="./images/Counter.png" alt="">
+        <h4>2015</h4> 
+    </div>
+     
+    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="tag">
+        <li><a href=""><button id="skill">html</button></a></li>
+        <li><a href=""><button id="skill">css</button></a></li>
+        <li><a href=""><button id="skill">javaScript</button></a></li>
+    </ul>
+    <button class="normal-button">See Project</button>
+</div>
+
+
+</div>
+    </section>
```

Comment:
I suggest you wrap the main section of your page ( the headline, work and about section) in a single `main` tag, as this makes your project more semantic😉

Reply:
@Ibizugbe I wrapped the main section in a single main tag as required